docs/guides/expectations/contributing/how-to-contribute-a-new-expectation-to-great-expectations.md

```diff
@@ -142,79 +144,103 @@ examples = [{
         }
     ],
 }]
-The value of examples is a list of examples.
+````
+
+The value of `examples` is a list of examples.
 
 Each example is a dictionary with two keys:
 
-data: defines the input data of the example as a table/data frame. In this example the table has one column named “mostly_threes” with 10 rows. If you define multiple columns, make sure that they have the same number of rows. If possible, include test data and tests that includes null values (None in the python test definition).
-tests: a list of test cases that use the data defined above as input to validate
+* `data`: defines the input data of the example as a table/data frame. In this example the table has one column named “mostly_threes” with 10 rows. If you define multiple columns, make sure that they have the same number of rows. If possible, include test data and tests that includes null values (None in the python test definition).
+
+* `tests`: a list of test cases that use the data defined above as input to validate
 
-title should be a descriptive name for the test case. Make sure to have no spaces.
+	* `title` should be a descriptive name for the test case. Make sure to have no spaces.
 include_in_gallery: set it to True if you want this test case to be visible in the gallery as an example (true for most test cases).
-in contains exactly the parameters that you want to pass in to the Expectation. "in": {"column": "mostly_threes", "mostly": 0.6} in the example above is equivalent to expect_column_values_to_equal_three(column="mostly_threes, mostly=0.6)
-out is based on the Validation Result returned when executing the Expectation.
-exact_match_out: if you set exact_match_out=False, then you don’t need to include all the elements of the result object - only the ones that are important to test.
+	* `in` contains exactly the parameters that you want to pass in to the Expectation. `"in": {"column": "mostly_threes", "mostly": 0.6}` in the example above is equivalent to `expect_column_values_to_equal_three(column="mostly_threes, mostly=0.6)`
+	* `out` is based on the Validation Result returned when executing the Expectation.
+	* `exact_match_out`: if you set `exact_match_out=False`, then you don’t need to include all the elements of the result object - only the ones that are important to test.
+
 Uncomment that code snippet and replace with your examples.
 
-Run run_diagnostics again. The newly added examples will appear in the output. They are not executed as tests yet, because most of the code in the Expectation is still commented out.
+Run `run_diagnostic`s again. The newly added examples will appear in the output. They are not executed as tests yet, because most of the code in the Expectation is still commented out.
 
-Note:
+:::note
```

Comment:
Ah this looks great! Exactly the formatting I was looking for when writing my own PRs.